db/db.connect.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const username = process.env['username'];
+const password = process.env['password'];
+
+const dbURI = `mongodb+srv://${username}:${password}@neog-cluster.wiph4.mongodb.net/videos`;
```

Comment:
URI can be stored in .env file using dotenv

Reply:
thanks, i should use env variable for URI also. for now im switching between two databases (one test db) so i will definitely add URI in env once i have finalized the database schema

also is dotenv a library? for now i am not using any library, repl.it and heroku both had env variable configuration. any advantages of dotenv?

Reply:
Yes, you're right.
As far as I know, while developing locally you should use dotenv so that you don't expose your API while pushing code on github.